widget/playground/src/containers/SupportedBlockchains/SupportedBlockchains.tsx

```diff
@@ -0,0 +1,64 @@
+import type { Type } from '../../types';
+
+import { ChainsIcon } from '@rango-dev/ui';
+import React from 'react';
+
+import { MultiSelect } from '../../components/MultiSelect/MultiSelect';
+import { getCategoryNetworks } from '../../helpers';
+import { useConfigStore } from '../../store/config';
+import { useMetaStore } from '../../store/meta';
+
+export function SupportedBlockchains({ type }: { type: Type }) {
+  const {
+    config: { from, to },
+    onChangeBlockChains,
+    onChangeBlockChain,
+    onChangeToken,
+  } = useConfigStore();
+  const {
+    meta: { blockchains },
+  } = useMetaStore();
+
+  const selectedType = type === 'Source' ? from : to;
+
+  const configBlockchains = selectedType?.blockchains;
+
+  const allBlockchain = blockchains.map((chain) => {
```

Comment:
`allBlockchains`. Please also use `blockchain` instead of `chain`.

Reply:
fixed.